tests/test.py

```diff
@@ -11,6 +11,20 @@ class Test:
     """
     A class for writing and running tests with a fluent API.
 
+    Attributes:
+        title (str): The title of the test.
+        mode (Literal["soft", "hard"]): The mode of the test. If set to "hard", the test will exit upon the first failure.
+        export_file_name (str): The name of the file where the test results will be exported.
+        result (str): The result of the test, formatted as a Markdown string.
```

Comment:
result (str): Test results, formatted as a Markdown string.

Reply:
Fixed in https://github.com/anand-kamble/data-filtering/pull/9/commits/d639c735d3a60db7d51c2bfc9c9cc1b74622c48c